notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb

```diff
@@ -0,0 +1,1202 @@
+{
```

Comment:
<p>Can you rephrase this to make it clearer that model.py gets included here, but pytorch gets included separately? maybe:</p><p> </p><pre class="ql-syntax" spellcheck="false">Create a <code>setup.py</code> script to bundle **model.py** in a tarball package. Notice that<code>setup.py</code> does not include the dependencies of <code>model.py</code> in the package. These dependencies are provided to your model version in other ways:

* <code>pandas</code> and <code>google-cloud-storage</code> are both included as part of [AI Platform Prediction runtime version 1.15](https://cloud.google.com/ai-platform/prediction/docs/runtime-version-list).
* <code>torch</code> is provided in a separate package, as described in a following section.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/105/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>ack</p>